package.json

```diff
@@ -6,7 +6,6 @@
   "main": "index.js",
   "scripts": {
     "build": "./bin/build.sh",
-    "build:vite": "vite build",
```

Comment:
removed all references to Vite. This is test code from Nathan Nelson that isnt being used